sos/plugins/rabbitmq.py

```diff
@@ -28,9 +28,25 @@ class RabbitMQ(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
     packages = ('rabbitmq-server',)
 
     def setup(self):
-        self.add_cmd_output("rabbitmqctl report")
-        self.add_cmd_output("rabbitmqctl cluster_status")
-        self.add_cmd_output("rabbitmqctl list_policies")
+        container_status = self.get_command_output(
+            "docker ps -a --format='{{ .Names }}'")
+
+        in_container = False
+        container_names = []
+        if container_status['status'] == 0:
+            for line in container_status['output'].splitlines():
+                if line.startswith("rabbitmq"):
+                    in_container = True
+                    container_names.append(line)
+
+        if in_container:
+            for container in container_names:
+                self.add_cmd_output('docker logs {0}'.format(container))
+                self.add_cmd_output(
+                    'docker exec -t {0} /bin/sh -c "rabbitmqctl report"'
+                    .format(container))
```

Comment:
The closing paren of the `self.add_cmd_output()` call should be on the following line, indented to the same level as the call.

Reply:
Ack, thanks